DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/system_console/jtag_avmm.tcl

```diff
@@ -0,0 +1,6 @@
+#  Copyright (c) 2022 Intel Corporation                                  
```

Comment:
Same remark: I hope that we can avoid shipping this file

Reply:
these system_console scripts should stay because they make it easier to test the design

Reply:
removed the system console script as well